.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: cpp
+
+compiler:
+- clang
+- gcc
+
+env:
+- QT_BASE=55
+
+before_install:
+- if [ "$QT_BASE" = "55" ]; then sudo add-apt-repository ppa:beineri/opt-qt55 -y; fi
```

Comment:
I would add Qt 5.4 to your tests as well since you have code that is Qt version specific, and also because righ tnow we are using Qt 5.4, not 5.5